### PR TITLE
Add new interlinks to blogposts

### PR DESCRIPTION
**Pull request by shadre**

### CODE REVIEW

File: _posts/2018-11-30-how-to-run-travis-ci-parallel-jobs-with-build-matrix-feature-fast.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ env:
     - KNAPSACK_PRO_CI_NODE_INDEX=1
 {% endhighlight %}
 
-How to run Cypress tests in JavaScript on parallel jobs with [@knapsack-pro/cypress](https://github.com/KnapsackPro/knapsack-pro-cypress):
+How to run Cypress tests in JavaScript on concurrent jobs with [@knapsack-pro/cypress](https://github.com/KnapsackPro/knapsack-pro-cypress):
```
Comment (ArturT) [inline _posts/2018-11-30-how-to-run-travis-ci-parallel-jobs-with-build-matrix-feature-fast.md:57]: what's the intention of use concurrent word? Just to have a new keyword in the article? I guess that's fine.
This reminded me about this what's the difference between concurrent and parallel https://softwareengineering.stackexchange.com/a/190725

File: _posts/2019-05-24-codeclimate-and-circleci-2-0-parallel-builds-config-for-rspec-with-simplecov-and-junit-formatter.md [hunk 2]
```diff
@@ -207,7 +207,7 @@ This example is based on [Knapsack Pro FAQ](https://knapsackpro.com/faq/question
 
 ## Summary and Queue Mode to do dynamic test suite split
 
-CI builds can be much faster thanks to leveraging parallel jobs on Circle CI 2.0 and CI parallelisation on any CI provider (<a href="/">see more parallelisation examples for your CI providers</a>). You can check <a href="https://knapsackpro.com?utm_source=docs_knapsackpro&utm_medium=blog_post&utm_campaign=codeclimate-and-circleci-2-0-parallel-builds-config-for-rspec-with-simplecov-and-junit-formatter">Knapsack Pro tool for CI parallelisation</a> and learn more about Queue Mode and what problems it solves in below video.
+CI builds can be much faster thanks to leveraging parallel jobs on Circle CI 2.0 and CI parallelisation on any CI provider (see more <a href="/continuous_integration">parallelisation examples for your CI providers</a>). You can check <a href="https://knapsackpro.com?utm_source=docs_knapsackpro&utm_medium=blog_post&utm_campaign=codeclimate-and-circleci-2-0-parallel-builds-config-for-rspec-with-simplecov-and-junit-formatter">Knapsack Pro tool for CI parallelisation</a> and learn more about Queue Mode and what problems it solves in below video.
```
Comment (ArturT) [inline _posts/2019-05-24-codeclimate-and-circleci-2-0-parallel-builds-config-for-rspec-with-simplecov-and-junit-formatter.md:210]: good catch with link /continuous_integration

File: _posts/2019-09-16-how-to-run-rspec-on-github-actions-for-ruby-on-rails-app-using-parallel-jobs.md [hunk 3]
```diff
@@ -114,8 +114,10 @@ I also recorded video showing how it all works and how CI builds with parallel j
 
 If you would like to better understand how Queue Mode works in Knapsack Pro and what else problems it solves you will find a few useful information in below video.
 
-<iframe width="560" height="315" src="https://www.youtube.com/embed/hUEB1XDKEFY" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+<iframe width="560" height="315" src="https://www.youtube.com/embed/hUEB1XKDEFY" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (ArturT) [inline _posts/2019-09-16-how-to-run-rspec-on-github-actions-for-ruby-on-rails-app-using-parallel-jobs.md:117]: you accidentally change url to invalid yt video. Here is correct one https://www.youtube.com/embed/hUEB1XDKEFY

File: _posts/2018-11-30-how-to-run-travis-ci-parallel-jobs-with-build-matrix-feature-fast.md (hunk 1 above)
Comment (shadre) [reply _posts/2018-11-30-how-to-run-travis-ci-parallel-jobs-with-build-matrix-feature-fast.md:57]: You are right it's technically not the same thing. I've seen there are some visits from a keyword like `travis concurrent jobs`, so I thought we can add this word here and there to improve the ranking on this specific keyword. The text may also read better when you don't duplicate the same word that much, but I'm fine with reverting this to keep it more technically accurate.

File: _posts/2019-09-16-how-to-run-rspec-on-github-actions-for-ruby-on-rails-app-using-parallel-jobs.md (hunk 3 above)
Comment (shadre) [reply _posts/2019-09-16-how-to-run-rspec-on-github-actions-for-ruby-on-rails-app-using-parallel-jobs.md:117]: :O Wow, great catch. Thanks

File: _posts/2018-11-30-how-to-run-travis-ci-parallel-jobs-with-build-matrix-feature-fast.md (hunk 1 above)
Comment (ArturT) [reply _posts/2018-11-30-how-to-run-travis-ci-parallel-jobs-with-build-matrix-feature-fast.md:57]: Let's keep concurrent. :)

**Pull request merged**